.travis.yml

```diff
@@ -23,6 +23,21 @@ matrix:
   allow_failures:
     - env: lt_branch=RC_1_0 gui=true build_system=cmake
     - env: lt_branch=RC_1_0 gui=false build_system=cmake
+  include:
+    - language: python
+      python: 2.7   # legacy Python
+      addons: true  # override
```

Comment:
Is this required? if not please remove it.

Reply:
It is required so that we do not slow down the testing by loading several dependencies that we do not need.